### PR TITLE
Handling of the new scoped-search CSS layout

### LINKED ISSUE #32: The search bar breaks in two lines when in a repository
Since today the GitHub search bar is breaking down in two lines when inside of a repository. Disabling this extension removes the problem. I guess GitHub changed something.

screen shot 2016-03-17 at 12 10 54

### PR DESCRIPTION
**Pull request by redox**
Fix #32

### CODE REVIEW

File: code/js/content.js [hunk 1]
```diff
@@ -289,8 +289,14 @@ function sanitize(text) {
 $(document).ready(function() {
   var $searchContainer = $('form#search_form, .site-search');
   var $q = $searchContainer.find('input[name="q"]');
-  var isRepository = $searchContainer.hasClass('repo-scope');
+  var isRepository = $searchContainer.find('.scope-badge').text() === 'This repository';
+  var isOrganization = $searchContainer.find('.scope-badge').text() === 'This organization';
```
Comment (redox) [inline code/js/content.js]: That's unfortunately the best I could think of for now to detect in which scope we are.

File: code/scss/content.sass [hunk 2]
```diff
@@ -74,13 +74,16 @@ $gh-primary-color: #3879D9
...
   .awesome-autocomplete
     .twitter-typeahead
       width: 230px
       .tt-dropdown-menu
         width: 360px
-        margin-left: -96px
+.site-search.repo-scope .awesome-autocomplete .twitter-typeahead .tt-dropdown-menu
+  margin-left: -96px
+.site-search.org-scope .awesome-autocomplete .twitter-typeahead .tt-dropdown-menu
+  margin-left: -109px
```
Comment (redox) [inline code/scss/content.sass]: Those hacks might be done nicely, happy to have your insights @LukyVj :+1:

File: code/js/content.js [hunk 3]
```diff
@@ -289,7 +289,16 @@ function sanitize(text) {
...
+  var $scopeBadge = $searchContainer.find('.scope-badge');
+  var $scopedSearch = $q.closest('.scoped-search');
+
+  function isRepository() {
+    return $scopeBadge.is(':visible') && $scopeBadge.text() === 'This repository';
+  }
+
+  function isOrganization() {
+    return $scopeBadge.is(':visible') && $scopeBadge.text() === 'This organization';
+  }
```
Comment (redox) [inline code/js/content.js]: That's unfortunately the best I could think of for now to detect in which scope we are.

File: code/scss/content.sass (hunk 2 above)
Comment (LukyVj) [inline code/scss/content.sass]: Seems good to me so far. 
The only bad side of this fix is to use fixed hardcoded values, the better way I'd suggest to do this is to get the actual width via Javascript : `$('.scope-badge').width()`

And to apply it to the `.twitter-typeahead` within a css `calc(100% - <extracted-width>)`.
Or to get the width of `.twitter-typeahead` in pixels & substract the width of the `.scope-badge` from it.

But for a quickfix, that works 🙌

**Pull request merged**